notebooks/official/training/distributed_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>Not sure if this is formatting issue in ReviewNb or there is actually a space in "customt rained".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>